app/src/main/java/org/systers/mentorship/view/activities/MemberProfileActivity.kt

```diff
@@ -22,7 +23,9 @@ class MemberProfileActivity : BaseActivity() {
     private val memberProfileViewModel by lazy {
         ViewModelProviders.of(this).get(MemberProfileViewModel::class.java)
     }
+    private lateinit var profileViewModel: ProfileViewModel
     private lateinit var userProfile: User
+    private lateinit var currentUser: User
```

Comment:
@mtreacy002 I don't see the `currentUser` variable being actually used, but only set. Is this what you want?

Reply:
@Isabel. currentUser is actually used to set the user who's logged in and want to send the request. It's being used in the line 43 and 112. This user is set using the ProfileViewModel.
Whereas the member whose the current user is going to send request to, is gonna be set using variable userProfile and the MemberProfileViewModel on line 59 and 115. Is this ok with you?